corehq/messaging/smsbackends/turn/forms.py

```diff
@@ -24,7 +24,7 @@ def gateway_specific_fields(self):
         )
         backend_choices = [('', _("No Fallback Backend"))]
         backend_choices.extend([
-            (backend.id, backend.name) for backend in domain_backends
+            (backend.couch_id, backend.name) for backend in domain_backends
```

Comment:
Because when sending the sms, it looks up the backend with the couch id

https://github.com/dimagi/commcare-hq/blob/a8c46b1d4fd453d1755cac4115519761960ffa7c/corehq/apps/sms/models.py#L258-L260